src/cssUsage.js

```diff
@@ -605,67 +605,80 @@ void function() { try {
 		/**
 		 * This will transform a value into an array of value identifiers
 		 */ 
-		function createValueArray(value, propertyName, dontNormalize = true) {
+		function createValueArray(value, propertyName, dontNormalize = false) {
 
 			// Trim value on the edges
 			value = value.trim();
 			
 			// Normalize letter-casing
 			value = value.toLowerCase();
 			
-			// Remove comments and !important
-			if(dontNormalize)
-			{
-				value = value.replace(/([/][*](?:.|\r|\n)*[*][/]|[!]important.*)/g,'');
-			}
-			
 			// Do the right thing in function of the property
-			if(dontNormalize)
-			{
-				switch(propertyName) {
+            if (!dontNormalize) {
+
+				// Remove comments and !important
+				value = value.replace(/([/][*](?:.|\r|\n)*[*][/]|[!]important.*)/g,'');

```

Comment:
This is very fragile.

Reply:
Yeah I don't like these regexes either, but I think this change would be outside the scope of this PR.